ansible/MANUAL_STEPS.md

```diff
@@ -479,3 +479,34 @@ After these steps are performed and the Pi's are running, Ansible can be run to
 [Setting up a Windows Host]: https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html
 [newer Ansible configuration]: https://github.com/nodejs/build/tree/master/ansible
 [stand-alone]: https://github.com/nodejs/build/tree/master/setup/windows
+
+## IBM i
+
+Ansible 2.7.6 is required.
+
+There isn't a system start service on IBMi -- the machine should not be
```

Comment:
This is out of date now, isn't it? the start is being submitted as some kind of IBMi specific service job?